sound/soc/intel/boards/Kconfig

```diff
@@ -545,7 +546,7 @@ if SND_SOC_SOF_ELKHARTLAKE
 
 config SND_SOC_INTEL_EHL_RT5660_MACH
 	tristate "EHL with RT5660 in I2S mode"
-	depends on I2C && ACPI
+	depends on I2C && ACPI && GPIOLIB
```

Comment:
Hmm, it this EHL going upstream as well. And if fact, do we have any sof-dev only machine drivers anymore. It's a bit painful to try to remember to check for this in patches like this (but you'll notice when sending upstream). I checked again and now with UP2 and SDW mach drivers upstream, we only seem to have cases like this which are just not sent yet...?  @plbossart

Reply:
I was thinking of sending this *after* providing the ELK machine driver upstream. There's no urgency for this patch, GPIOLIB should be enabled in most distros, it's more of a paranioa check